src/sagemaker/rl/estimator.py

```diff
@@ -277,6 +281,18 @@ def train_image(self):
         """
         if self.image_name:
             return self.image_name
+
+        # use different account for rl images if ray version is later than 0.8.2
```

Comment:
coach > 1.0.0 also has the different account/repo, right? (looking at https://github.com/aws/sagemaker-rl-container/#rl-images-provided-by-sagemaker)